Panels/TabbedPanels.js

```diff
@@ -182,7 +182,12 @@ const TabbedPanelsBase = kind({
 			if (tabs) {
 				return tabs.map((tab, index) => {
 					const {view: View, viewProps} = tab;
-					return <View key={index} {...viewProps} />;
+					const tabView = <View key={index} {...viewProps} />;
+
+					delete tab.view;
+					delete tab.viewProps;
+
+					return tabView;
```

Comment:
Sorry but I did not find the case that occured console.warning 
Could you please share the code so that i see the diffrence?



> This feature allows TabbedPanel to create its views from the tabs prop if there it has no children. The problem stems from the fact that it did not delete the attributes necessary to create those viewes before transmitting the tabs prop to TabGroup(which does not support those attributes) and this did result in console warnings.

I tried with

```
		<TabbedPanels
				{...rest}
				index={panelIndex}
				tabs={[
					{view: 'div', title: 'Button', icon: 'netbook'},
					{view: 'div', title: 'Item', icon: 'aircirculation'},
					{view: 'div', title: 'LabeledIconButton', icon: 'temperature'}
				]}
			/>
```

But i didn't see any diffrence.

Reply:
The stucture I used was this one:  

<TabbedPanels
		orientation={'vertical'}
		tabs={[
			{title: 'Panel1', icon: 'netbook', view: Panel, viewProps:{children: 'Panel 1'}}
		]}
/>

Reply:
I cleaned up the code a bit by adding two "delete".